src/constants/fixed_bases/value_commit_v.rs

```diff
@@ -803,13 +802,13 @@ mod tests {
     }
 
     #[test]
-    fn lagrange_coeffs_short() {
+    fn lagrange_coeffs() {
```

Comment:
This did not need to be moved back into `orchard` from (what will go into) `halo2_gadgets`, since it is testing code that is provided by `halo2_gadgets`, not constants hard-coded in `orchard`. That being said, if we in future wanted to hard-code the Lagrange coefficients, we _would_ need these tests here, so I don't mind `test_lagrange_coeffs` being in the public testing API of `halo2_gadgets`.